services/kubecost/metadata.yaml

```diff
@@ -2,7 +2,7 @@ displayName: Kubecost
 description: Provides real-time cost visibility and insights for teams using Kubernetes, helping organizations continuously reduce cloud costs.
 category:
   - monitoring
-type: partner
+type: platform
```

Comment:
Why is this changing?

Reply:
https://nutanix.slack.com/archives/C06A78Y063B/p1734635813445609?thread_ts=1734623086.853099&cid=C06A78Y063B